static/js/pages/SitesDashboard.tsx

```diff
@@ -73,7 +73,7 @@ export default function SitesDashboard(): JSX.Element {
         <StudioList>
           {listing.results.map((site: Website) => (
             <StudioListItem
-              title={site.title}
+              title={`${site.title} (${site.short_id})`}
               subtitle={siteDescription(site) ?? ""}
```

Comment:
I think it would be good to visually differentiate the title / short id somehow. An easy way to do this would be to use the subtitle prop on `StudioListItem`:
```suggestion
              title={site.title}
              subtitle={site.short_id}
```

Interesting... I just took a quick look at `siteDescription(site)` which is supposed to display the course number and term, one of the goals of this short_id feature. But `siteDescription(site)` uses outdated metadata properties, so it never does anything. Safe to remove.

<img width="918" alt="Screen Shot 2022-05-16 at 1 09 50 PM" src="https://user-images.githubusercontent.com/9010790/168648255-6f8bb909-2d85-480b-a05c-a9688569043e.png">

Reply:
I agree. Subtitle is certainly more appropriate and a good visual differentiator. I was just a bit unsure whether it's fine to remove the existing metadata or not but thanks for confirming. I'll update the PR.